packages/mobile/src/screens/product-info/components/image-carousel/image-carousel.tsx

```diff
@@ -0,0 +1,35 @@
+import { ProductDto } from '@vse-bude/shared';
+import React, { FC } from 'react';
+import { ListRenderItem } from 'react-native';
+import { Image, Text, View, FlatList } from '~/components/components';
+import { globalStyles } from '~/styles/styles';
+import { styles } from './styles';
+
+const ImageCarousel: FC<Pick<ProductDto, 'imageLinks'>> = ({ imageLinks }) => {
+  const renderItem: ListRenderItem<string> = ({ item, index }) => (
+    <View style={styles.imgContainer}>
+      <Image source={{ uri: item }} style={styles.image} />
+      <View style={styles.itemsCount}>
+        <Text style={styles.currentItem}>{(index + 1).toString()}</Text>
+        <Text
+          style={styles.totalItems}
+        >{`/${imageLinks.length.toString()}`}</Text>
```

Comment:
`${imageLinks.length.toString()}`` - Extract this to a separate variable. And there is no need to call `toString()`

Reply:
fixed